From 1b3dc6cdf9c49cc109532577b2e93dfe62c8dc21 Mon Sep 17 00:00:00 2001
From: Finley Xiao <finley.xiao@rock-chips.com>
Date: Mon, 27 Aug 2018 17:51:59 +0800
Subject: [PATCH] Revert "Revert "arm64: dts: rockchip: rk3308-evb-v10: Add
 regulator-early-min-microvolt for core""

This reverts commit 46d3be8026baf79c87e81eae1a7faa42ff76b944.

This patch add a new 'regulator-early-min-microvolt' property to limit
the minimal voltage of regulator during kernel startup.

Change-Id: I580f0cceda280b9168e7f489f39785830cd28a28
Signed-off-by: Finley Xiao <finley.xiao@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3308-evb-v10.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3308-evb-v10.dtsi b/arch/arm64/boot/dts/rockchip/rk3308-evb-v10.dtsi
index 40f44013fe5b..339e0330b5f8 100644
--- a/arch/arm64/boot/dts/rockchip/rk3308-evb-v10.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3308-evb-v10.dtsi
@@ -154,6 +154,7 @@
 		regulator-min-microvolt = <827000>;
 		regulator-max-microvolt = <1340000>;
 		regulator-init-microvolt = <1015000>;
+		regulator-early-min-microvolt = <1015000>;
 		regulator-always-on;
 		regulator-boot-on;
 		regulator-settling-time-up-us = <250>;
-- 
2.35.3

